### PR TITLE
implement merge function

### LINKED ISSUE #383: Fix merge method
There is currently a method called `merge` that lives in `/lib/merge.js`.
It is incomplete and needs to be fixed!

`merge` combines multiple objects into a single one with all the keys of each object. When two objects have the same key, the value from the second object is used.

### Acceptance Criteria:
1. Running `yarn test merge` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by danswater**
Closes #383

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: lib/get.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+* `get` the value of a prop inside an object
+* if value is undefined return default
+*
+* @param {Objects} object to inspect
+* @param {String} property name
```
Comment (ktilcu) [inline lib/get.js]: Should we have get take a path? like `'a.b.c'` or should we separate to 2 functions? `prop` and `path`. Should the default value be optional? What do you think @danswater

Comment (danswater) [reply lib/get.js]: right now I'm thinking `path` should be a simple property name, if we we're going to take a path like this `'a.b.c'` then we should also consider this `a[0].b.c`. And yeah you're right default value should be optional I'll update the test case.

Comment (danswater) [reply lib/get.js]: @ktilcu just let me know if you guys would like path to support this scenario e.g `'a.b.c'`, `'a[0].b.c'` or `'a.b[0].c'` etc... and I will update the test case for that scenario.
